src/query/service/src/servers/http/v1/query/http_query.rs

```diff
@@ -64,6 +66,19 @@ pub struct HttpQueryRequest {
     pub stage_attachment: Option<StageAttachmentConf>,
 }
 
+impl Debug for HttpQueryRequest {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_struct("HttpQueryRequest")
+            .field("session_id", &self.session_id)
+            .field("session", &self.session)
+            .field("sql", &short_sql(self.sql.clone()))
```

Comment:
could `short_sql` sanitize the sql's sensitive content like access secrets?

Reply:
no, the sql is not parsed and we don't what content is sensitive.